scheduler/drilling-wits-depth-median/src/api.py

```diff
@@ -0,0 +1,75 @@
+from typing import List
+
+import pydantic
+from corva import Api
+
+from src import models
+from src.configuration import SETTINGS
+
+
+def get_records(
+    api: Api,
+    asset_id: int,
+    log_identifier: str,
+    top_depth: float,
+    bottom_depth: float,
+) -> List[models.Record]:
+    raw_records = api.get_dataset(
+        provider='corva',
+        dataset=SETTINGS.IN_DATASET,
+        query={
+            'asset_id': asset_id,
+            'log_identifier': log_identifier,
+            'measured_depth': {'$gt': top_depth, '$lte': bottom_depth},
```

Comment:
Should this be `measured_depth: {'$lte': top_depth, '$gt': bottom_depth}`?

Reply:
top depth is always smaller than bottom depth, but thanks for vigilance

Reply:
```python3
"""
top_depth: start depth in ft., covered by this event. Use exclusively.
bottom_depth: end depth in ft., covered by this event. Use inclusively.
"""
```